gallery.html

```diff
@@ -4,8 +4,24 @@
     <meta charset="UTF-8">
```

Comment:
This lacked creativity and innovation of which I KNOW you are capable.  It's certainly a clean file, but could do with a bit more styling for position and appeal.